### PR TITLE
New issue templates.

### PR DESCRIPTION
**Pull request by lgritz**
GitHub lets you create issue templates, in fact multiple kinds, so
when somebody files an issue they see a menu of the categories. Each
category can have a different template.

I'm making four categories: (1) bug report, (2) feature request, (3)
build problem, and (4) question.

I'm also making it very clear for (3) and (4) that they should not
file an issue at all, but rather use the oiio-dev mail list.

Hopefully this will direct people toward filing more helpful bug reports and
feature requests, and redirect the usual build troubles or usage questions
to the mail list where they belong.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+A clear and concise description of what you expected to happen.
+
+**Evidence**
+Do you have error messages? (please quote exacty) Screenshots? Example
+command lines or scripts that reliably reproduce the error? If it only
+happens with certain image files, can you attach the smallest image you
+can make that reproduces the problem?
+
+**Platform information:**
+ - OIIO branch/versionn:
```
Comment (fpsunflower) [inline .github/ISSUE_TEMPLATE/bug_report.md]: typo: `version`

Comment (lgritz) [reply .github/ISSUE_TEMPLATE/bug_report.md]: good catch, fixed

**Comment by fpsunflower** [comment]
LGTM!

**Comment by lgritz** [comment]
Merged. I guess we'll see if this is effective in improving bug reports and directing people with simple build problems or questions to the mail list instead of filing issues.

**Pull request merged**